src/routes/api/v2/users.js

```diff
@@ -318,7 +318,9 @@ router.put("/favorites/:course_id", checkJwt, async (req, res, next) => {
       res.status(400).send({ message: "Course is already in favorites." });
       return;
     }
-    const course = await prisma.courses.findUnique({ where: { id: course_id } });
+    const course = await prisma.courses.findUnique({
```

Comment:
這邊是neo的api
是排版的時候動到了
是否就保持原樣?

Reply:
Just leave it. It's okay since you are going to create a new repo in the future.
But remember to wrap SQL execution in try-catch blocks in your new repo.

Reply:
My question was "wrapping the SQL execution" instead of the change itself. No matter if that is from the Neo API, if there is anything that can be improved, just refactor it right away. Please wrap the SQL execution in the next PR.